Clarify current status of the clients, v2 and coverage breadth (#245)
bblfsh/documentation: README.md

### Current status

#### Universal AST coverage

> Not all the constructs are converted in a language-independent way yet.

Reviewer: ```suggestion
Not all constructs are converted in a language-independent way yet.
```

> As of 1Q2019, every language driver is expected to only support Identifiers,

Reviewer: ```suggestion
As of Q1 2019, every language driver is expected to support Identifiers,
```

> String (literals), Imports, Functions in the [Universal AST schema](uast/uast-specification-v2.md).

Reviewer: ```suggestion
String literals, Imports, and Functions in the [Universal AST schema](uast/uast-specification-v2.md).
```

> That is almost everything that let's one to examine the symbols exported

Reviewer: ```suggestion
That is sufficient to examine the symbols exported by the package, but not control flow (yet).
```

> by the package, but not the control flow (yet).

Reviewer: ```suggestion
```

> Meanwhile, in order to locate constracts that are not covered by UAST

Reviewer: ```suggestion
Meanwhile, in order to identify constructs that are not covered by the UAST
```

schema yet - one can look for a `@role` field.
A [Role](https://godoc.org/gopkg.in/bblfsh/sdk.v2/uast/role#Role) is added
to every native AST node and it contains a language-independent annotations.

> Underlying tree structure for a constract will be different between languages though.

Reviewer: ```suggestion
However, the tree structure for a role construct may be different between languages.
```

Reviewer: I was trying to think of a good, short example we could show to illustrate this, but I couldn't come up with anything that didn't generate half a page of tree.

#### Protocol v2

> Currently, Babelfish is in the process of transition to [v2 protocol](uast/uast-specification-v2.md),

Reviewer: ```suggestion

Currently, Babelfish is in the process of transition to an updated data model, denoted [UAST v2](uast/uast-specification-v2.md),
```

> new node representation and Semantic UAST.

Reviewer: ```suggestion
which includes a new node representation and canonicalized ("semantic") UAST structures.
```

> All the beta+ drivers support it in the latest version

Reviewer: ```suggestion
All the beta+ drivers support UAST v2 in their latest versions. UAST v2 support requires bblfshd ≥ 2.6.1.
```

> and requires bblfshd >= 2.6.1.

Reviewer: ```suggestion
```

> Libuast was updated to support the new node format, but some of the

Reviewer: ```suggestion
Libuast has been updated to support the new node format, but some of the
```
